views/default/js/elgg/admin.js

```diff
@@ -0,0 +1,186 @@
+/**
+ * Admin-area specific javascript functions.
+ *
+ * @since 1.8
+ */
+define(function(require) {
+	var $ = require('jquery');
+	var ui = require('jquery-ui');
+	var elgg = require('elgg');
+
+	function init () {
+		// system messages do not fade in admin area, instead slide up when clicked
+		$('.elgg-system-messages li').stop(true);
+		$('.elgg-system-messages li').die('click');
+		$('.elgg-system-messages li').live('click', function() {
+			$(this).stop().slideUp('medium');
+		});
+
+		// draggable plugin reordering
+		$('#elgg-plugin-list > ul').sortable({
```

Comment:
This was broken in the commit this was based on. Can you rebase master and re-push?